xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/client/accounts/AccountTransactionsRequestParams.java

```diff
@@ -38,10 +38,70 @@
    *
    * @return An {@link ImmutableAccountTransactionsRequestParams.Builder}.
    */
+  @Deprecated
   static ImmutableAccountTransactionsRequestParams.Builder builder() {
     return ImmutableAccountTransactionsRequestParams.builder();
   }
 
+  /**
+   * Construct a builder for this class with {@link LedgerIndex} field set.
+   *
+   * @param ledgerIndex An index of type {@link LedgerIndex} to specify a particular ledger.
+   * @return An {@link ImmutableAccountTransactionsRequestParams.Builder}.
+   */
+  static ImmutableAccountTransactionsRequestParams.Builder builder(LedgerIndex ledgerIndex) {
+    return ImmutableAccountTransactionsRequestParams.builder()
+        .ledgerSpecifier(
+        Optional.of(LedgerSpecifier.of(ledgerIndex))
+    );
+  }
+
+  /**
+   * Construct a builder for this class with {@link LedgerSpecifier} field set.
+   *
+   * @param ledgerSpecifier A specifier of type {@link LedgerSpecifier} to specify a ledger.
+   * @return An {@link ImmutableAccountTransactionsRequestParams.Builder}.
+   */
+  static ImmutableAccountTransactionsRequestParams.Builder builder(LedgerSpecifier ledgerSpecifier) {
+    return ImmutableAccountTransactionsRequestParams.builder().ledgerSpecifier(Optional.ofNullable(ledgerSpecifier));
+  }
+
+  /**
+   * Construct a builder for this class with ledgerHash field set.
+   *
+   * @param ledgerHash Hash of type {@link Hash256} of a specific ledger.
+   * @return An {@link ImmutableAccountTransactionsRequestParams.Builder}.
+   */
+  static ImmutableAccountTransactionsRequestParams.Builder builder(Hash256 ledgerHash) {
```

Comment:
I don't think we want this builder available here. The contract we want to prod developers to be using is only `LedgerSpecifier`. If someone wants to specify a hash, then they should build a LedgerSpecifier and supply it above above on line 65.

Reply:
Do we want to have the similar thing for LedgerIndex? Even that seems to be passed via LedgerSpecifier.